charts/rancher-gke-operator/1.0.200/Chart.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
```

Comment:
For new charts in system-charts we do a first commit that is just a copy of old chart, and then 2nd commit that has any changes, that way you can view the real diff. Changes here would just be around version and such.

Reply:
done